lib/falcon/service/server.rb

```diff
@@ -63,6 +64,16 @@ def setup(container)
 						
 						instance.ready!
 						
+						if health_check_timeout
+							Async(transient: true) do
```

Comment:
TIL, transient: that's handy. 

For the curious: https://github.com/socketry/async/tree/main/guides/asynchronous-tasks#transient-tasks